File: src/__tests__/golden/fixtures/python-flask/app/services/user_service.py
from ..models.user import User
from .auth import hash_password


def get_user(user_id):
    return User(user_id, "Test", "test@example.com")


def create_user(name, email, password):
    hashed = hash_password(password)
    return User("new", name, email)

File: src/__tests__/golden/fixtures/java-service/src/main/java/com/example/service/UserService.java
package com.example.service;

import com.example.model.User;
import com.example.repository.UserRepository;

public class UserService {
    private final UserRepository repository = new UserRepository();

    public User getUser(Long id) {
        return repository.findById(id);
    }

    public User createUser(String name, String email) {
        User user = new User(name, email);
        repository.save(user);
        return user;
    }
}
